atom/browser/api/atom_api_window.cc

```diff
@@ -424,6 +424,16 @@ void Window::ShowDefinitionForSelection() {
 }
 #endif
 
+#if defined(OS_MACOSX) || defined(OS_LINUX)
```

Comment:
I think we should expose them on Windows.

There are some APIs not exposed on all platforms, but I feel they were bad decisions.